plugins/modules/files/iso_get_files.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Ansible Project
+# Copyright: (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_get_files
+short_description: Fetch files from ISO
+description:
+    - This module is used to get specific files from ISO.
+author:
+    - Yuhua Zou (@ZouYuhua) <zouy@vmware.com>
+requirements:
+- "pycdlib"
+- "python >= 2.7"
+version_added: '5.4.0'
```

Comment:
The 5.4.0 release is tomorrow and this will definitely not make it in that release (not enough time to properly review it otherwise), so feel free to bump the version to 5.5.0 already :-)

Reply:
Thanks. Done